src/ui/component/app/view.jsx

```diff
@@ -64,12 +66,12 @@ class App extends React.PureComponent<Props> {
   enhance: ?any;
 
   render() {
-    const { enhancedLayout } = this.props;
+    const { enhancedLayout, language } = this.props;
 
     return (
       <div id="window" onContextMenu={e => openContextMenu(e)}>
-        <Header />
-        <SideBar />
+        <Header language={language} />
+        <SideBar language={language} />
```

Comment:
@seanyesmunt I did this to trigger re-renders when the language changes. Is this the proper way to do this [given the `__` function is not a React component]?

Reply:
At this moment menu not work :)

Reply:
@kodxana it should now

Reply:
Discussed on the standup, but I would move these to `{header, sideBar}/index.js`